src/idl_gen_dart.cpp

```diff
@@ -181,7 +181,6 @@ class DartGenerator : public BaseGenerator {
     }
 
     auto &code = *code_ptr;
-    if (indent) code += indent;
```

Comment:
@dnfield can you review these changes?
I have removed one extra indent for the first comment line.

Reply:
I'm not sure I fully understand this, but I don't see what it would end up hurting. How does the sample output change with this?

Reply:
Before the fix, the first line of comments has a double indent:
https://github.com/google/flatbuffers/blob/8d86b5347fbeba85f11b7bf479334f6700feb455/src/idl_gen_dart.cpp#L183-L190.
The first added before `for` loop, second inside the loop.
This double-indent was introduced by #4676.

Current fix removes the first (out of the loop) indentation.
Changes in the generated code (tests/monster_test_my_game.example_generated.dart):
https://github.com/google/flatbuffers/pull/5346/commits/8d85aa52dac56f372e5622b9149885dcb56971b6

Reply:
Ahh thanks for the explanation.  This LGTM then!

Reply:
Thanks